manifests/base/repo-server/argocd-repo-server-deployment.yaml

```diff
@@ -301,11 +301,11 @@ spec:
         - mountPath: /home/argocd/cmp-server/plugins
           name: plugins
       initContainers:
-      - command:
-        - /bin/cp
-        - -n
-        - /usr/local/bin/argocd
-        - /var/run/argocd/argocd-cmp-server
+      - args:
+          - /bin/cp -n /usr/local/bin/argocd /var/run/argocd/argocd && /bin/ln -s /var/run/argocd/argocd /var/run/argocd/argocd-cmp-server
```

Comment:
wouldn't the symlink command fail if the symlink is already present? Secondly, if the `cp` step is skipped but file exists, the symlink command might cause 'file exists' error.

Reply:
On startup this is an empty dir so those files do will not exists, the cp command can not bet skipped because this is all one command so if cp fails the whole command would fail.